satpy/readers/__init__.py

```diff
@@ -546,7 +546,9 @@ class FSFile(os.PathLike):
     def __init__(self, file, fs=None):
         """Initialise the FSFile instance.
 
-        *file* can be string or an fsspec.OpenFile instance. In the latter case, the follow argument *fs* has no effect.
+        *file* can be string, an object implementing the PathLike protocol, or
+            an fsspec.OpenFile instance. In the latter case, the follow argument
+            *fs* has no effect.
```

Comment:
This should just be switched to an `Args:` list. I also prefer two backticks around literals over the asterisks for bold (I know you didn't do it, but still).

Reply:
Fixed.